core/order.go

```diff
@@ -739,14 +809,35 @@ func validateCryptocurrencyOrderItem(item *pb.Order_Item) error {
 	return nil
 }
 
+func (n *OpenBazaarNode) getDivisibility(code string) uint32 {
+	dict := repo.LoadCurrencyDefinitions()
+	defn, err := dict.Lookup(code)
+	if err != nil {
+		return 0
```

Comment:
We might want to log a warning when we're hitting this. Something like: `log.Warningf("unable to find currency code (%s) in local dictionary")`.